src/main/java/org/jenkins/tools/test/model/PluginCompatReport.java

```diff
@@ -180,9 +179,10 @@ public static PluginCompatReport fromXml(File reportPath) throws IOException {
         return report;
     }
 
-    private static XStream2 createXStream(){
-        XStream2 xstream = new XStream2(new Xpp3DomDriver());
+    private static XStream createXStream(){
+        XStream xstream = new XStream(new MXParserDomDriver());
         xstream.setMode(XStream.NO_REFERENCES);
+        xstream.allowTypes(new Class[] {MavenCoordinates.class, PluginCompatReport.class, PluginCompatResult.class, PluginInfos.class});
```

Comment:
Fixing a regression apparently introduced by one of my previous PRs that I noticed while doing manual testing: newer versiosn of XStream need non-basic types to be explicitly allowed. While I was here I also switched the driver to the standard recommended XStream driver. Jenkins has its own complicated reasons for using a custom driver, but they don't apply here. Anyway, I hope to rip out this XStream stuff in the not-so-distant future, so don't pay too much attention to it.